django_mongodb/utils.py

```diff
@@ -25,6 +26,39 @@ def check_django_compatability():
         )
 
 
+def parse_uri(uri, conn_max_age=0, conn_health_checks=False, test=None):
+    """
+    Convert the given uri into a dictionary suitable for Django's DATABASES
+    setting.
+    """
+    uri = parse_uri_mongo(uri)
+    host = None
+    port = None
+    if uri["fqdn"] is not None:
+        # If the fqdn is present, this is a SRV URI and the host is the fqdn.
+        host = f"mongodb+srv://{uri['fqdn']}"
+    else:
+        nodelist = uri.get("nodelist")
+        if len(nodelist) == 1:
+            host, port = nodelist[0]
+        elif len(nodelist) > 1:
+            host = ",".join([f"{host}:{port}" for host, port in nodelist])
+    settings_dict = {
+        "ENGINE": "django_mongodb",
+        "NAME": uri["database"],
```

Comment:
```suggestion
        "NAME": name or uri["database"] or '',
```

Reply:
I would say let's just not add the key 1519e0f484682f2ad7b89bbb670c0c9c9fac0498